### PR TITLE
Initial printing support for sd-export-usb

### PR DESCRIPTION
**Pull request by emkll**
#### Functional testing

Note that for now this should only work with brother printers due to .ppd configuration that must be done within the dispVM itself. I will try to get my hands on other printer makes/models to  support other vendors.

Testing should be very similar to that performed in #259, as the handler script is the same and the export archive format is the same
- Specify persistent usb device in `config.json` (you can see which usb device is used by running `qvm-usb` in dom0)
- Provision sd-export vms by either running `make all` or `make sd-export` with a printer connected to the configured usb
In sd-svs:
- create a .sd-export archive with export method in `metadata.json` set to `"device": "printer-test"`
- qvm-open-in-vm sd-export-usb <archive>.sd-export
- [ ] Printer settings appear, printer is selectable
- [ ] Printer test page prints successfully
- create a .sd-export archive with export method in `metadata.json` set to `"device": "printer"`
- add some files under the export_data folder to be printed
- qvm-open-in-vm sd-export-usb <archive>.sd-export
- [ ] All files in export_data folder are printed
- [ ] No files remain in /tmp/

#### App tests (for sd-export-usb)
- create python3 vitualenv and install requirements
- pytest sd-export/test-export.py
- [ ] All tests pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,7 +182,11 @@ qvm-copy-to-vm sd-export-usb ~/.securedrop_client/data/name-of-file
 
 The development plan is to provide functionality in the *SecureDrop Client* that automates step 3, and assists the user in taking these steps via GUI prompts. Eventually we plan to provide other methods for export, such as [OnionShare](https://onionshare.org/) (this will require the attachment of a NetVM), using a dedicated export VM template with tools such as OnionShare and Veracrypt. The next section includes instructions to approximate the OnionShare sharing flow.
 
-#####  Automated export flow (Work in progress, client integration TBD)
+##### Automated export flows
+
+The `sd-export-usb` disposable VM handles exports to USB devices through `qvm-open-in-vm`. USB device IDs are configured in `config.json`. The automated export flows make use of the `qvm-usb --persistent` feature. This means that the persistent USB device must be available for `sd-export-usb` to start. In other words, a USB memory stick or a printer must be connected **prior** to the the `qvm-open-in-vm sd-export-usb` call is made.
```
Comment (eloquence) [inline README.md]: Nit: "before the" instead of "prior to the the"

**Comment by eloquence** [comment]
I understand that we'll likely want to treat each print job as blocking other print jobs for now. I'm still curious what happens in this proof-of-concept implementation if we don't. 

What happens in this implementation if you kick off a job when one is already in progress?  And one after that? Does the printer queue get cleared out? Do previous jobs get interrupted?

**Comment by emkll** [comment]
> What happens in this implementation if you kick off a job when one is already in progress? And one after that? Does the printer queue get cleared out? Do previous jobs get interrupted?

No : because the Disposable VM is not shut down, the previous job remains in the printer queue, and sent to the printer. If another job is triggered, it will appended to the queue and be printed after the first one. Given the manual nature of the export flow at this point, it's very hard to effectively test for race conditions.

**Comment by zenmonkeykstop** [comment]
Followed instructions as given for setup.
**manual testing**
On `sd-svs`:
Created a .sd-export archive with export method in metadata.json set to "device": "printer-test" and ran `qvm-open-in-vm sd-export-usb .sd-export`
- [x] Printer settings appear, printer is selectable *(PPD for specific printer not available but default Brother br7030.ppd worked)*
- [x] Printer test page prints successfully

Created a .sd-export archive with export method in metadata.json set to "device": "printer" and added pdf and text files under the export_data folder to be printed, ran `qvm-open-in-vm sd-export-usb .sd-export`
- [x] All files in export_data folder are printed
- [x] No *related* files remain in /tmp/

**app tests**
Set up python3 virtualenv on `sd-export-usb` as follows:
```
sudo apt-get install python3-pip
sudo apt-get install python3-virtualenv
sudo apt-get install git
git clone -b 267-printing https://github.com/freedomofpress/securedrop-workstation && cd securedrop-workstation
python3 -m virtualenv -p python3 .
source ./bin/activate
pip install -r sd-export/test-requirements.txt
pytest -v sd-export/test_export.py
```
All 12 tests passed, 1 warning about the imp module's deprecation.

Looks good @emkll!

**Pull request merged**